requirements.txt

```diff
@@ -0,0 +1,8 @@
+arrow
+bleach
+fedmsg[all]
```

Comment:
For the others we just used ``fedmsg`` would it make sense to do that here as well?